.github/workflows/tics.yml

```diff
@@ -31,9 +31,8 @@ jobs:
 
     - name: Install build dependencies
       run: |
-        sudo apt-get update
-        sudo apt-get install -y devscripts equivs lcov
-        mk-build-deps -s sudo -i
+        sudo --preserve-env apt-get install --yes devscripts equivs lcov ninja-build
+        sudo --preserve-env apt-get build-dep --yes ./
```

Comment:
Is this approach preferable to `mk-builds-deps`? Only here or in general? Why?

Reply:
Again, how Mir was doing it. Tested and it doesn't make a difference either way. Reverting to stay consistent with the Multipass `README.md`.

Reply:
Got it, thanks.

Reply:
It's just avoiding a step, using `apt` itself rather than an intermediary.

Using `mk-build-deps` makes more sense on user systems, as it makes it easier to uninstall. But in CI, it's just a waste.